### ISSUE TITLE
jaccard_similarity_score is not what we want

### ISSUE THREAD

**Issue by dan-bishopfox** [BishopFox/eyeballer#26]
jaccard_similarity_score
https://scikit-learn.org/stable/modules/generated/sklearn.metrics.jaccard_similarity_score.html#sklearn.metrics.jaccard_similarity_score

This is not the same thing as our overall binary accuracy. (We're a multilabel classifier. The page even gives a warning: "In binary and multiclass classification, this function is equivalent to the accuracy_score. It differs in the multilabel classification problem.")

I... don't see a sklearn equivalent for what we want though?
